src/Setup/DevDivPackages/Debugger/DevDivPackagesDebugger_DoNotUse.csproj

```diff
@@ -22,15 +22,5 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.DiaSymReader.PortablePdb" Version="$(MicrosoftDiaSymReaderPortablePdbVersion)" />
```

Comment:
Does this file really need to exist now, given it's referencing just a single thing we also reference?

Reply:
Nope, will remove it